Update the certbot documentation to better utilize the updated certbot versions and information (#4200)
mattermost/docs: source/install/config-proxy-nginx.rst

SSL and HTTP/2 with server push are enabled in the provided configuration example.

.. note::

>   If you're going to use Let's Encrypt to manage your SSL certificate stop at step 3 here and please see the `Nginx HTTP/2 & SSL full configuration guide <https://docs.mattermost.com/install/config-ssl-http2-nginx.html>`__.

Reviewer: ```suggestion
  If you're going to use Let's Encrypt to manage your SSL certificate stop at step 3 here and please see the `NGINX HTTP/2 & SSL full configuration guide <https://docs.mattermost.com/install/config-ssl-http2-nginx.html>`__.
```

.. note::

>   You will need valid SSL certificates in order for NGINX to pin the certificates properly. Additionally, your browser must have permissions to accept the certificate as a valid CA-signed certificate.

Reviewer: ```suggestion
  You'll need valid SSL certificates in order for NGINX to pin the certificates properly. Additionally, your browser must have permissions to accept the certificate as a valid CA-signed certificate.
```
